sleuth_crawler/scraper/scraper/pipelines.py

```diff
@@ -5,6 +5,8 @@
 # Don't forget to add your pipeline to the ITEM_PIPELINES setting
 # See: http://doc.scrapy.org/en/latest/topics/item-pipeline.html
 import json
+from sleuth_backend.solr_module.solr_module import add_item
```

Comment:
I think we should rename `solr_module` to just `solr`, since having a module name with "module" in it is redundant.